lucene/analysis/morfologik/src/java/org/apache/lucene/analysis/uk/UkrainianMorfologikAnalyzer.java

```diff
@@ -145,7 +152,7 @@ protected TokenStreamComponents createComponents(String fieldName) {
 
   private static Dictionary getDictionary() {
     try {
-      return Dictionary.read(UkrainianMorfologikAnalyzer.class.getResource("ukrainian.dict"));
+      return Dictionary.read(UkrainianMorfologikAnalyzer.class.getResource("/ua/net/nlp/ukrainian.dict"));
```

Comment:
I am a bit confused on why for class loader getResource() we need relative path (surprisingly the full one starting with "/" does not work for me), but otherwise it makes sense so I'll push the fix shortly.